server/controllers/cohort.js

```diff
@@ -206,7 +250,7 @@ const deleteCohortById = (req, resp) => {
     user: { _id: userId }
   } = req;
   const sanitizedCohortId = sanitize(cohortId);
-  let cohortMembers;
+  let cohort;
```

Comment:
use `returnPayload` helper

Reply:
In this case, I can't use returnPayload, because I have to rebuild this helper and refactor all its usage. In this case, I need cohort data and data return by the promise I passed to this helper.